frontend_app_thread/src/container/PopupCreateThread.jsx

```diff
@@ -83,7 +84,7 @@ class PopupCreateThread extends React.Component {
   handleChangeNewContentName = e => this.setState({newContentName: e.target.value})
 
   handleClose = () => GLOBAL_dispatchEvent({
-    type: 'hide_popupCreateContent', // handled by tracim_front:dist/index.html
+    type: CUSTOM_EVENT.HIDE_POPUP_CREATE_CONTENT, // handled by tracim_front:dist/index.html
```

Comment:
now we have a variable, it is easier to find where are the usages of a given custom event.
So you can remove this kind of comments